### PR TITLE
Write commitment

### PR DESCRIPTION
**Pull request by blckngm**

### CODE REVIEW

File: axon/src/message.rs [hunk 1]
```diff
@@ -173,10 +175,25 @@ pub struct MsgChannelCloseConfirm {
...
+    }
+}
+
 // As our CKB convention, the content of the packet is stored in Witness.
 // We don't need to place it again in this message.
 #[derive(RlpDecodable, RlpEncodable)]
-pub struct MsgSendPacket {}
+pub struct MsgSendPacket {
+    // keccak256(commitment path), keccak256(commitment value)
+    pub commitments: Vec<CommitmentKV>,
```
Comment (ashuralyk) [inline axon/src/message.rs]: where are these commitments stored?

Comment (blckngm) [reply axon/src/message.rs]: They are not “stored”. Each transaction will have one or maybe a few commitments. They are only present in the witness of the particular transaction.

**Review by ashuralyk — APPROVED** [review]

File: axon/src/handler/objects.rs [hunk 2]
```diff
@@ -65,10 +67,24 @@ impl Default for IbcChannel {
...
+            state: value.state.proto_channel_state() as i32,
+            ordering: proto::channel::Order::from(value.order) as i32,
+        }
+    }
+}
+
 #[derive(RlpEncodable, RlpDecodable, Debug, Clone, PartialEq, Eq)]
 pub struct IbcPacket {
     pub packet: Packet,
-    pub tx_hash: Option<H256>,
```
Comment (ashuralyk) [inline axon/src/handler/objects.rs:71]: why this field removed? it's in need for now in relayer

File: axon/src/lib.rs [hunk 3]
```diff
@@ -213,8 +223,34 @@ pub fn connection_id(client_id: &str, connection_idx: usize) -> String {
     )
 }
 
-pub fn get_channel_id_str(idx: u16) -> String {
```
Comment (ashuralyk) [inline axon/src/lib.rs:216]: this function is also in demand for now in relayer to generate channel id

File: axon/src/handler/objects.rs (hunk 2 above)
Comment (ashuralyk) [reply axon/src/handler/objects.rs:71]: this can be solved in relayer

File: axon/src/lib.rs (hunk 3 above)
Comment (ashuralyk) [reply axon/src/lib.rs:216]: this can also be solved in relayer

**Pull request merged**